lib/include/secw_snmpv3.h

```diff
@@ -50,6 +53,8 @@ enum Snmpv3PrivProtocol : uint8_t
 {
     DES = 0,
     AES,
+    AES256,
+    AES192 = 4,
```

Comment:
why 4? Any conflicts? And natural order seems be better... AES192, AES256

Reply:
was requested as requirement to have this order